### DISCUSSION TITLE
markdown block in comments render werid

### DISCUSSION THREAD

**Discussion by lf-shaw** [requarks/wiki#6341] [Error / Bug Report]
markdown comment contains block such as 

```python
def a():
    print('hello')
```

and 

```
$$
\alpha
$$
```

render 

![image](https://user-images.githubusercontent.com/16026142/231094218-01d9ef35-1a00-43a5-a618-9f57b375a526.png)

system information

![image](https://user-images.githubusercontent.com/16026142/231094563-e4413f39-b670-425a-adcc-726cc2466d80.png)

**Comment by NGPixel** [comment]
The comments are meant to be used with basic markdown, not complex syntax like code blocks or math expressions. Although code blocks support could be added in the future.

**Reply by lf-shaw** [reply]
thanks a lot!

**Comment by wowexec** [comment]
Can I alter some codes to make "$$" supported in the comment?

**Comment by petrowsky** [comment]
For anyone else who finds this issue and wants to have code blocks within comments emulated like the prismjs based code (without line numbers and language support - visual only), you can adapt the following to your needs. It would be nice if WikiJS allowed for the Default comments provider to have a config option to apply the prismjs to \<code\> blocks within comments. Add the following to **Theme > Code Injection > CSS Override**

```
.comments-post-content pre {
  border-radius: 5px;
  background-color: #000;
  padding: 12px;
  overflow: auto;
}
.comments-post-content pre code {
  padding: 0 !important;
}
```
